asapdiscovery-data/asapdiscovery/data/backend/openeye.py

```diff
@@ -839,10 +894,44 @@ def get_SD_data(mol: oechem.OEMol) -> dict[str, str]:
     Dict[str, str]
         Dictionary of SD data
     """
-    sd_data = {dp.GetTag(): dp.GetValue() for dp in oechem.OEGetSDDataPairs(mol)}
+    from collections import defaultdict
+
+    sd_data = defaultdict(list)
+    for conf in mol.GetConfs():
+        for dp in oechem.OEGetSDDataPairs(conf):
+            sd_data[dp.GetTag()].append(dp.GetValue())
     return sd_data
 
 
+def get_SD_data(mol: oechem.OEMolBase) -> dict[str, str]:
```

Comment:
I think it should be possible to abstract away from the user  whether we need to use a `get_multiconf` or `get_` regular. Is this the case? Same for the `set` methods. 

IE it would be great if we didn't have to worry as a user to select the correct method. Does this seem feasible.